portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+let old = -1;
```

Comment:
Is it possible to get current index without global variable?

Reply:
I will try if I can wrap it in OOP object

Reply:
Fixed